src/librustdoc/html/static/main.js

```diff
@@ -2621,6 +2624,16 @@ function defocusSearchBar() {
         });
     }());
 
+    function showImportantTraits(content) {
+        let list = content.classList
+    }
+
+    onEachLazy(document.getElementsByClassName("important-traits"), function(e) {
+        e.onclick = function() {
+            e.getElementsByClassName('important-traits-tooltiptext')[0].classList.toggle("force-tooltip")
```

Comment:
I think it's better to use either `this` or `event.target` in here (`event` would be the first argument given to onclick).

Reply:
Wouldn't that just be `e` anyway?

Reply:
I'm not too confident about the variable capture process in JS so I prefer to minimize it as much as possible.